docs/content/Coordinator.md

```diff
@@ -3,3 +3,286 @@ title: this page has moved.
 layout: simple_page
 redirect_to: design/coordinator.html
 ---
+<<<<<<< HEAD
```

Comment:
Yes they were polluted in this PR along the way and I'm in the process of cleaning it up.